packages/react/src/FeatureFlags/DefaultFeatureFlags.ts

```diff
@@ -8,4 +8,5 @@ export const DefaultFeatureFlags = FeatureFlagScope.create({
   primer_react_overlay_overflow: false,
   primer_react_segmented_control_tooltip: false,
   primer_react_select_panel_fullscreen_on_narrow: false,
+  primer_react_select_panel_order_selected_at_top: true,
```

Comment:
any issues with having a FF on by default? 🤔

Reply:
Nope! Should be okay 👍